retention/.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = venv
+max-line-length = 120
```

Comment:
Just a style nit - but ideally this project should follow the same conventions as other matrix python projects. Synapse uses black for formatting which defaults to 88 characters and I would propose running black against this (as a separate PR).

Reply:
Ack. Have installed black now (but not committed its run) and updated flake8 config to match (52b2609)